assets/compat/config_spec.yaml

```diff
@@ -115,21 +115,23 @@ bitcoind:
   name: Bitcoin Core
   description: |
     The Bitcoin Core node to connect to:
+      - none: Use the light bitcoin backend built into LND, Neutrino.
       - internal: The Bitcoin Core and Proxy services installed to your Embassy
-      - external: An unpruned Bitcoin Core node running on a different device
   tag:
     id: type
     name: Type
     variant-names:
+      none: None (Built-in LND Neutrino)
       internal: Internal (Bitcoin Core)
       internal-proxy: Internal (Bitcoin Proxy)
-      external: External
     description: |
       The Bitcoin Core node to connect to:
-        - internal: The Bitcoin Core and Proxy services installed to your Embassy
-        - external: An unpruned Bitcoin Core node running on a different device
+        - none: Use the light bitcoin backend built into LND, Neutrino. If using Neutrino, please switch to using Bitcoin Core as soon as possible. Neutrino uses the BIP157/8 light client protocol, which has security risks.
```

Comment:
"Please switch to using Bitcoin Core as soon as possible." is odd given this is an explicit, opt-in selection. Maybe "It is recommended to run LND against Bitcoin. Neutrino uses..."